pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -239,15 +239,13 @@ message ConnectionState {
 
 // DeploymentInfo contains information relevant to an application deployment
 message DeploymentInfo {
-  repeated ComponentParameter params = 1;
-
-  optional string revision = 2;
+  optional string revision = 1;
 
-  repeated ComponentParameter componentParameterOverrides = 3;
+  repeated ComponentParameter componentParameterOverrides = 2;
 
-  optional k8s.io.apimachinery.pkg.apis.meta.v1.Time deployedAt = 4;
+  optional k8s.io.apimachinery.pkg.apis.meta.v1.Time deployedAt = 3;
 
-  optional int64 id = 5;
+  optional int64 id = 4;
```

Comment:
It's not safe to renumber. Just remove the field and let the others keep their existing numbers.